.github/nopejs.yml

```diff
@@ -0,0 +1,36 @@
+name: Express CI
```

Comment:
This yml file needs to also go inside a folder called workflows so it will look like `.github/workflows/nodejs.yml`